nerdlets/github-about/setup.js

```diff
@@ -191,6 +194,31 @@ export default class Setup extends React.PureComponent {
             <a href="https://github.com">https://github.com</a> or it could be a
             private GitHub enterprise instance.
           </p>
+          {!githubUrl &&
+            accountGithubUrl &&
+            accountGithubUrl !== '' &&
```

Comment:
if `accountGithubUrl` is `''` than it will evaluate to false in the line above.

```suggestion
```

Reply:
ooh yes thanks